src/themes/components/articleTeaser.ts

```diff
@@ -0,0 +1,19 @@
+import { ComponentMultiStyleConfig, SystemStyleObject } from '@chakra-ui/react';
+
+const title: SystemStyleObject = {
+  textStyle: 'heading3',
+  color: 'blue.700',
+};
+const text: SystemStyleObject = {
+  textStyle: 'base',
```

Comment:
Is it possible to add the color text here as well? -> `color: gray.900`

Reply:
Hey 👋 I found something else...
The text doesn't apply our font custom because we don't have any  `base`  typography token.

<img width="1099" alt="Screenshot 2022-07-28 at 09 07 39" src="https://user-images.githubusercontent.com/37380787/181443351-f1960e72-f73d-49d7-ab97-3f4acf61c8b3.png">

Reply:
🤦🏻  true it's `body` not `base`...